.github/workflows/build.yml

```diff
@@ -15,15 +15,11 @@ jobs:
     strategy:
       matrix:
         platform: [ ubuntu-latest, macos-latest, windows-latest ]
-        python-version: [ 3.6, 3.7, 3.8, 3.9, "3.10" ]
-        exclude:
-          - platform: macos-latest
+        python-version: [ 3.8, 3.9, "3.10" ]
+        include:
+          - platform: ubuntu-20.04
             python-version: 3.6
-          - platform: macos-latest
-            python-version: 3.7
```

Comment:
Why drop macos and windows?

Reply:
This inverses `exclude` to `include` so we can specify `ubuntu-20.04`